config/config_defaults.go

```diff
@@ -931,6 +932,16 @@ const CGRATES_CFG_JSON = `
 },
 
 
+"dispatcherh":{
+    "enabled": false,
+    "dispatchers_conns": [],
+	"host_ids": {},  
```

Comment:
Was it not agreed here that we go with config per tenant? Something like{ "*default": {}, "cgrates.org":{}} maybe?